fahrschein-example/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>org.zalando</groupId>
         <artifactId>fahrschein-parent</artifactId>
-        <version>0.20.0-SNAPSHOT</version>
+        <version>0.19.0</version>
```

Comment:
I think we can remove the `<version>` tag from all submodules, and they will inherit from the parent.

Reply:
Lets consider an extra issue for that.